### PR TITLE
[PT-BR] Mimicry Additional Upgrades and Minor Fixes on Brazilian Translation.

### PR DESCRIPTION
**Pull request by Firething**
# **SCP:SL Translation PR Template**

## **Language Name**
Brazilian Portuguese

## **State Type of change (New language, changed existing Etc.)**
Fixed Steam_StorePage.json (formatting error), Minor grammar fixes, Changed existing (SCP939_HUD, LoadingHints...)

# **Checklist:**
- [x] I have tested the translation changes ingame
- [x] I have made sure to not commit changes to multiple translations at the same time, or non-translation repo files
- [x] I have ran `EncodingNormalizer.exe` on my changed files before commiting 
   -  *( Exception to this is when edits has __only__ been made in-browser. If this is the case, also* ***check this box*** *)*
- [x] I have made sure that I'm **not** submitting a duplicate translation 
- [x] I have made sure my addition follows guidelines described in the [CONTRIBUTING.md](https://github.com/northwood-studios/SCPSL-Translations/blob/master/.github/CONTRIBUTING.md) file
- [x] I have made sure to label my translation folder by [Microsoft Language Tag Standards](https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-lcid/a9eac961-e77d-41a6-90a5-ce1a8b0cdb9c)
   - Only in cases of **new** translations (Can be tested [Here](https://rextester.com/WDXPS97501)). Also check this box if the change is **not** a new translation
> For more information regarding game translations, please visit our [README](https://github.com/northwood-studios/SCPSL-Translations/blob/master/README.md) page.

### CODE REVIEW

File: Translations/pt-BR/Items.txt [hunk 1]
```diff
@@ -22,13 +22,13 @@
...
 25~Granada Altamente-Explosiva~Boom!
-26~Granada Flashbang~Cega seus inimigos.
+26~Granada de Luz~Cega seus inimigos.
 30~COM-18
 31~SCP-018~Superbola capaz de quicar com extrema eficiência.
 32~SCP-268~Quando usado, torna você temporariamente invisível, à menos que seja interrompido por qualquer interação do usuário.
 33~Adrenalina~Aumenta temporariamente a resistência.
 34~Analgésicos~Trata machucados menores.
-35~Moeda~Você não tem certeza por que deseja carregar isso por aí...
+35~Moeda~Você não tem certeza do por que deseja carregar isso por aí...
```
Comment (Farinz) [inline Translations/pt-BR/Items.txt:31]: Correction sugestion:

35-Moeda~Você não tem certeza se deseja carregar isso aí...

File: Translations/pt-BR/Keybinds.txt [hunk 2]
```diff
@@ -35,4 +35,4 @@ Bloquear Porta
 Liberar Todos Bloqueios
 Blecaute
 Confinamento
-Ping
+Indicação
```
Comment (Farinz) [inline Translations/pt-BR/Keybinds.txt:38]: Indicação is not wrong but I think Ping is more widely used.

**Comment by Red-Z** [comment]
@Firething had my friend @Farinz look over it, just look at the changes and we're good

**Comment by Firething** [comment]
@Red-Z took a look and made some changes based on some corrections. Was also thinking of leaving "Ping" untranslated, but I believe translating it makes the purpose of the ability more apparent

**Review by Red-Z — APPROVED** [review]

**Pull request merged**